spacy/tests/pipeline/test_edit_tree_lemmatizer.py

```diff
@@ -0,0 +1,280 @@
+import pickle
+import pytest
+from hypothesis import given
+import hypothesis.strategies as st
+from spacy import util
+from spacy.lang.en import English
+from spacy.language import Language
+from spacy.pipeline.edit_trees import EditTrees
+from spacy.training import Example
+from spacy.strings import StringStore
+from spacy.util import make_tempdir
```

Comment:
These could now be relative imports.

Reply:
The test suite intentionally uses absolute imports so it behaves the same when you run the tests on the installed package.